sysproduction/interactive_manual_check_historical_prices.py

```diff
@@ -29,11 +29,15 @@ def interactive_manual_check_historical_prices():
     :return: Nothing
     """
     with dataBlob(log_name="Update-Historical-prices-manually") as data:
-        instrument_code = get_valid_instrument_code_from_user(data, source='single')
-        check_instrument_ok_for_broker(data, instrument_code)
-        data.log.label(instrument_code = instrument_code)
-        update_historical_prices_with_checks_for_instrument(
-            instrument_code, data)
+        do_another = True
+        while do_another:
+            instrument_code = get_valid_instrument_code_from_user(data, source='single')
+            check_instrument_ok_for_broker(data, instrument_code)
+            data.log.label(instrument_code=instrument_code)
+            update_historical_prices_with_checks_for_instrument(instrument_code, data)
+            ans = input("Another <y> ? or <RETURN> to exit: ")
```

Comment:
"Another <type anything> or RETURN to exit"

Reply:
Are you asking me to change the prompt?

Reply:
No need, I've already done it. Thanks for the pull, good idea.